vault/logical_system.go

```diff
@@ -607,11 +609,28 @@ func (b *SystemBackend) handleRekeyRetrieve(
 		return logical.ErrorResponse("no backed-up keys found"), nil
 	}
 
+	keysB64 := map[string][]string{}
```

Comment:
We don't want to store the keysB64 from RekeyBackup and retrieve them back, is it?
Instead of populating it while returning?